smart_selects/static/smart-selects/admin/js/bindfields.js

```diff
@@ -61,10 +61,11 @@
         initItem(chained);
     }
 
-    $(document).on('formset:added', function (event, $row, formsetName) {
+    $(document).on('formset:added', function (event) {
         // Fired every time a new inline formset is created
 
         var chainedFK, chainedM2M, filteredM2M;
+        var $row = $(event.target);
```

Comment:
Something like this will allow backward compatibility

```suggestion
    $(document).on('formset:added', function (event, _row) {
        // Fired every time a new inline formset is created

        var chainedFK, chainedM2M, filteredM2M;
        var $row = _row || $(event.target);
```